docs/REST API/rest-api.md

```diff
@@ -3986,28 +3992,62 @@ List all configuration profiles for macOS hosts enrolled to Fleet's MDM that are
       "profile_id": 1337,
       "team_id": 0,
       "name": "Example profile",
+      "platform": "darwin",
       "identifier": "com.example.profile",
       "created_at": "2023-03-31T00:00:00Z",
       "updated_at": "2023-03-31T00:00:00Z",
       "checksum": "dGVzdAo="
+    },
+    {
+      "profile_id": 1338,
+      "team_id": 0,
+      "name": "Example Windows profile",
+      "platform": "windows",
+      "created_at": "2023-04-31T00:00:00Z",
+      "updated_at": "2023-04-31T00:00:00Z",
+      "checksum": "aCLemVr)"
     }
   ]
 }
 ```
 
-### Download custom macOS setting (configuration profile)
+### Get or download custom OS setting (configuration profile)
 
-`GET /api/v1/fleet/mdm/apple/profiles/{profile_id}`
+> [Download custom macOS setting](https://github.com/fleetdm/fleet/blob/ee02782eaf84c121256d73abc20b949d31bf2e57/docs/REST%20API/rest-api.md#download-custom-macos-setting-configuration-profile) (`GET /api/v1/fleet/mdm/apple/profiles/{profile_id}`) API endpoint is deprecated as of Fleet 4.40. It is maintained for backwards compatibility. Please use the API endpoint.
+
+`GET /api/v1/fleet/mdm/profiles/{profile_id}`
```

Comment:
Do we use the `{profile_id}` format elsewhere in the docs? Or do we just use `{id}`. I think we want to be consistent here.

Reply:
Good point. Seems that we don't use this format anywhere else. For scripts and queries we use just `{id}`